koschei/backend/services/resolver.py

```diff
@@ -464,12 +464,12 @@ def generate_repo(self, sack, collection, repo_id):
             .filter_by(collection_id=collection.id)\
             .delete()
         prev_state = collection.state_string
+        collection.latest_repo_id = repo_id
```

Comment:
There's a problem with atomicity - resolver issues commit for each batch of packages processed. So the repo is set as resolved before all the packages is resolved (only first batch). It would be better if both latest_repo_id and latest_repo_resolved were set before the last commit (as latest_repo_id was set before).